packages/app/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@backstage/cli/config/tsconfig.json",
+  "include": ["src"],
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    "outDir": "../../dist-types/packages/app",
```

Comment:
Sadly `yarn build` (i.e. `backstage-cli package build`) requires `dist-types` to be at the root level.